test/unit/argument_parser/format_parse_validators_test.cpp

```diff
@@ -399,7 +452,8 @@ TEST(validator_test, output_directory)
         const char * argv[] = {"./argument_parser_test", "-h"};
         seqan3::argument_parser parser{"test_parser", 2, argv, false};
         test_accessor::set_terminal_width(parser, 80);
-        parser.add_positional_option(path, "desc", seqan3::output_directory_validator{});
+        parser.add_positional_option(path, "desc",
+                                     seqan3::output_directory_validator{});
```

Comment:
Did you forget to add the mode?

Reply:
Ah no, this is the output_DIRECTORY_validator. I dont know why I added a line break. 😅